src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/c0fd8031c3.js" defer crossorigin="anonymous"></script>
+    <title>To Do List</title>
+  </head>
+  <body>
+    <h1>My Todo List</h1>
+    <div id="todo-list-container">
+      <div id="todo-day-container">
+        <p>Today's To Do</p>
+        <div id="refresh"></div>
+      </div>
+      <form id="add-task">
+        <input id="add-task-text" type="text" maxlength="100" placeholder="Add to your list ..." />
+        <img id="enter-Button" draggable="false" />
+      </form>
```

Comment:
You should add two placeholders one for the author and the second for the book title. It will show your code more professionally.